src/Xrefcheck/Config.hs

```diff
@@ -95,6 +105,7 @@ overrideConfig config
     defScanners = cScanners $ defConfig flavor
     defExclusions = cExclusions $ defConfig flavor
     defNetworking = cNetworking $ defConfig flavor
+    defRedirectConfig = []
```

Comment:
Oh, I now see why my proposal was not so clear, till this moment we defaulted fields to the values from the default config.

Reply:
Yeah! I though that it should still being that way, but it is clear now.